modules/edit-modes/src/lib/measure-distance-mode.ts

```diff
@@ -1,137 +1,180 @@
-// @ts-ignore
 import turfDistance from '@turf/distance';
-// @ts-ignore
+import { FeatureCollection } from '../geojson-types';
+import { ClickEvent, PointerMoveEvent, ModeProps, GuideFeatureCollection, Tooltip } from '../types';
+import { getPickedEditHandle } from '../utils';
 import memoize from '../memoize';
-import {
-  ClickEvent,
-  PointerMoveEvent,
-  Tooltip,
-  ModeProps,
-  GuideFeatureCollection,
-  EditHandleFeature,
-} from '../types';
-import { FeatureCollection, Position } from '../geojson-types';
 import { GeoJsonEditMode } from './geojson-edit-mode';
 
-const DEFAULT_TOOLTIPS = [];
-
 export class MeasureDistanceMode extends GeoJsonEditMode {
```

Comment:
It would be great to see the tests updated for this enhancement

Reply:
I've added test coverage to updated mode. The strange thing here is that before all tests for edit-modes and for some other parts of nebula were switched off in jest config file (lib folder was added to exclusions and all test under it were off). I enabled them and after it appeared that some of them were broken. I was able to manage to fix a few but there is another part that uses DeckGl lib and requires headless gl context execution (I believe that in previous versions DeckGl supported headless mode so they were working fine before DeckGl dropped this feature). I've tried to fix them also but unfortunately i still receiving strange errors with WebGl context execution, so finally i decided to switch them off (it's only 2 tests, not too much) and try to figure out how to repair them later. The rest of the enabled testsuites are working just fine now

Reply:
I don't think deck.gl has dropped this feature, but maybe some upgrade of headless-gl is needed. Since you are enabling more tests, it should be fine to switch them off for now, I would recommend just adding an issue to track that they need to be restored. @supersonicclay

Reply:
Thank you for fixing those tests @andrei-kucherov!